generators/app/templates/mvc/controllers/robots.txt.js

```diff
@@ -1,15 +1,8 @@
-const fs = require('fs')
+const path = require('path')
 
 module.exports = function (app) {
   app.route('/robots.txt').get(function (req, res) {
     res.setHeader('Content-Type', 'text/plain')
-
-    // it's plain text, so we don't want to render it with the template parser
-    fs.readFile(app.get('viewsPath') + 'robots.txt', 'utf8', function (err, data) {
-      if (err) {
-        console.error(err)
-      }
-      res.send(data)
-    })
+    res.sendFile(path.join(app.get('viewsPath'), 'robots.txt'))
```

Comment:
cache `path.join(app.get('viewsPath'), 'robots.txt')` into a variable outside of the module.exports